components/dashboard/src/Login.tsx

```diff
@@ -57,7 +57,7 @@ export function Login() {
             await openAuthorizeWindow({
                 login: true,
                 host,
-                onSuccess: () => updateUser(),
+                onSuccess: (payload?: string) => authorizeSuccessful(payload),
```

Comment:
minor: declaring the function after it is used seems a bit uncommon to me. Also, the value can be used directly as the signature matches (i.e. `onSuccess: authorizeSuccessful`)

Reply:
Thanks - updated & moved